src/processing/vFramer/include/vDraw.h

```diff
@@ -171,8 +205,27 @@ class skinsampleDraw : public vDraw {
 
 class flowDraw : public vDraw {
 
+protected:
+    int Xlimit;
```

Comment:
this is outside the scope of the vDraw - and is also doing something incorrectly